doc/configuration.rst

```diff
@@ -481,18 +481,24 @@ We recommend that you copy this set of exit codes to your ``luigi.cfg`` file:
   missing_data=20
   task_failed=30
   scheduling_error=35
+  unknown_reason=38 or 0
   unhandled_exception=40
 
 unhandled_exception
   For internal Luigi errors.  Defaults to 4, since this type of error
   probably will not recover over time.
+unknown_reason
+  For when a task does not run successfully because of an unknown reason. Despite
+  this case can be expected in an error free execution, it does not guarantee
+  completeness of the root task. Return code 38 is advised only for cases in which
+  Luigi return code 0 is used by the user to guarantee root tasks' completeness.
```

Comment:
Indeed, I was going to suggest 25. (As said, for a human operator, "not run for unknown reason" IMO evaluates worse than "not run for \<insert known reason, like missing data>".)